marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/flow/updateFlow.api

```diff
@@ -16,7 +16,7 @@
             "desc": "The order of stepIds can be updated.",
             "datatype": "array",
             "nullable": true,
-            "$javaClass": "com.fasterxml.jackson.databind.node.JsonNode"
+            "$javaClass": "com.fasterxml.jackson.databind.node.ArrayNode"
```

Comment:
Does this actually work? I thought DS only accepts JsonNode. 

To test it out, run "../gradlew generateFlowInterface" from ./marklogic-data-hub

Reply:
It seems to accept ArrayNode as well https://docs.marklogic.com/guide/java/DataServices#id_pgfId-1081275 . It's a good idea to ensure "../gradlew generateFlowInterface" from ./marklogic-data-hub runs fine.

Reply:
../gradlew generateFlowInterface didn't like JsonNode as a datatype for stepIds, which is why I made the change to ArrayNode. Does JsonNode work as a datatype for stepIds for you both? @rjrudin @srinathgit

Reply:
../gradlew generateFlowInterface fails in current develop. It runs fine after changing updateFlow.api from JsonNode to ArrayNode

Reply:
Yeah just noticed a few things- the datatype doesn't match for a javaclass JsonNode. The javaclass is also incorrect- should be sans the node in databind.node. I need a few minutes to figure out if I want this to be an array or a json doc.

Reply:
I think it makes more sense for it to be an array, keeping these changes.